pkg/options/resource.go

```diff
@@ -27,6 +27,7 @@ var (
 	// DefaultResources represents the default set of resources in kube-state-metrics.
 	DefaultResources = ResourceSet{
 		"certificatesigningrequests":      struct{}{},
+		"clusterrolebindings":             struct{}{},
```

Comment:
I wonder if we should really have these exposed by default. The use cases for this kind of metric seem really niche to me and I wouldn't want to add even more timeseries to the default setup if it is not really useful to the majority of ksm users. Same comment can be applied to the recently added `role` and `clusterrole` since they are not part of a release yet, we can still afford to remove them from the default if deemed meaningful.

Any thoughts on that @fpetkovski @mrueg?

Reply:
I think not exposing them by default makes sense. I agree that the amount of metrics has crept up over time and we can do a better job at providing sane defaults.

Reply:
sounds good to have standard set, should we resolve that in a follow-up PR since it will affect other resources as well?

Reply:
So for this PR, should I remove this line?

Reply:
No all good, we will continue in a follow up PR!